src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -191,7 +191,10 @@ class ObjectsTableUI extends Component {
         });
       }
       toastNotifications.addDanger({
-        title: `Unable find saved objects`,
+        title: this.props.intl.formatMessage({
```

Comment:
Please define `intl` during restructuring of `this.props` here:
https://github.com/elastic/kibana/blob/44a7dcbf138c62e8f423a87a74aa20292e6eb134/src/legacy/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js#L167

```js
const { intl, savedObjectsClient } = this.props;
```

And use `intl` directly:
```suggestion
        title: intl.formatMessage({
```